cmd/lotus-shed/ledger.go

```diff
@@ -291,3 +293,72 @@ var ledgerShowCmd = &cli.Command{
 		return nil
 	},
 }
+
+var ledgerNewAddressesCmd = &cli.Command{
+	Name:  "new",
+	Flags: []cli.Flag{},
+	Action: func(cctx *cli.Context) error {
+		ctx := lcli.ReqContext(cctx)
+
+		if cctx.NArg() != 1 {
+			return fmt.Errorf("must pass account index")
+		}
+
+		index, err := strconv.Atoi(cctx.Args().First())
```

Comment:
CodeQL probably wants you to use
```suggestion
		index, err := strconv.ParseUint(cctx.Args().First(), 10, 32)
```

Reply:
done